src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -371,24 +184,26 @@ function getStateFromUpdate(update, instance, prevState, props) {
   }
 }
 
-function beginUpdateQueue(
+function beginUpdateQueue<State>(
```

Comment:
I never really liked this name. :/ I keep forgetting what it does.

What does it mean to begin an update queue?